.pipelines/Common.yml

```diff
@@ -77,6 +77,109 @@ stages:
         official: ${{ parameters.official }}
         rid: osx-x64
 
+  - job: packages_windows
```

Comment:
>packages_windows [](start = 9, length = 16)

Packages have to be signed on a Windows container (limitations of the current infra), but tests have to run on the right OS as separate jobs. This is different than the GH build.